test/utils/test_cost_funcs.py

```diff
@@ -170,3 +170,27 @@ def test_cost_func_dd(self):
         )[0]
 
         self.assertLess(best_layout[1], best_layout2[1])
+
+    # def test_cost_func_ecr(self):
```

Comment:
Just delete the code instead of commenting it out. Maybe add a TODO to add a test once mapomatic works with a fake Sherbrooke backend.